.gitignore

```diff
@@ -201,15 +201,19 @@ fabric.properties
 .idea/httpRequests
 
 # custom file configuration
-empty_environment_variables.py
-empty_test_environment_variables.py
-filled_func_environment_variables.py
-filled_perf_environment_variables.py
-install_perf_environment_variables.py
+/benchmark_runner/main/empty_environment_variables.py
+/benchmark_runner/main/empty_test_environment_variables.py
+/benchmark_runner/main/filled_func_environment_variables.py
+/benchmark_runner/main/filled_perf_environment_variables.py
+/benchmark_runner/main/install_perf_environment_variables.py
+/benchmark_runner/main/install_func_environment_variables.py
+/benchmark_runner/main/operator_perf_environment_variables.py
+/benchmark_runner/main/operator_func_environment_variables.py
```

Comment:
Why are these being ignored?  If they're temporary files, they should be in a temporary directory.

Reply:
I prefer to keep it there because it simplify my debuging, I also suggest that you hold those files on your end, at least
/benchmark_runner/main/filled_func_environment_variables.py

Reply:
Yeah, I really don't like that because it adds technical debt, but let's try to take mutable stuff out of tree, which would let us run multiple concurrent things from the same source tree.